ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/Record.java

```diff
@@ -61,7 +61,7 @@ public Record(
     this.topic = topic;
     this.key = key;
     this.value = value;
-    this.jsonValue = jsonValue;
+    this.jsonValue = Optional.ofNullable(jsonValue);
```

Comment:
You can just make the param `Optional<JsonNode>` - Jackson knows how to handle them. Do this on both constructors and you'll simplify the code later on that's doing `someOptional.orElse(null)`.

Reply:
this isn't the jackson `RecordNode` - this is the resolved node